twisted/internet/defer.py

```diff
@@ -47,9 +49,12 @@ class CancelledError(Exception):
 
 class TimeoutError(Exception):
```

Comment:
I think that TimeoutError should inherit from internet.error.TimeoutError in this way existing code trying to catch defer.TimeoutError will continue to work as espected and will only get a warning